samples/bluetooth/fast_pair/locator_tag/configuration/boards/nrf54l15dk_nrf54l05_cpuapp.conf

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=3584
```

Comment:
We could consider optimizing stacks for Oberon use-case (unless we are going to switch to CRACEN in near future)

Reply:
we have plenty of RAM still left. I don't want to focus on RAM optimizations for now.